### PR TITLE
Suppress cryptography warnings for paramiko

### PR DESCRIPTION
**Pull request by mattclay**
##### SUMMARY

Suppress cryptography warnings for paramiko.

##### ISSUE TYPE

Feature Pull Request

### CODE REVIEW

**Comment by jborean93** [comment]
Is this even worthwhile as they have merged a change that uses the new imports https://github.com/paramiko/paramiko/pull/2421.

**Comment by mattclay** [comment]
I think it's still worthwhile, for anyone who ends up using a version of paramiko without the changes.

**Comment by mattclay** [comment]
I can update the comment though, to indicate the version which fixes the issue.

File: lib/ansible/module_utils/compat/paramiko.py [hunk 1]
```diff
@@ -11,7 +11,12 @@
 
 try:
     with warnings.catch_warnings():
-        warnings.filterwarnings('ignore', message='Blowfish has been deprecated', category=UserWarning)
+        # Blowfish is deprecated in cryptography, but still used by paramiko versions older than 2.9.5.
+        # See: https://github.com/paramiko/paramiko/pull/2039
+        warnings.filterwarnings('ignore', message='Blowfish has been ', category=UserWarning)
+        # TripleDES is deprecated in cryptography, but still used by paramiko versions older than 3.3.2 and 3.4.1.
```
Comment (jborean93) [inline lib/ansible/module_utils/compat/paramiko.py]: Being pedantic it hasn't been deprecated, it's just moved to a new namespace and a newer version of Cryptography will remove the old import path in the new version.

**Review by jborean93 — APPROVED** [review]
Up to you if you wish to clarify the comment but the change looks good

File: lib/ansible/module_utils/compat/paramiko.py (hunk 1 above)
Comment (mattclay) [reply lib/ansible/module_utils/compat/paramiko.py]: I revised the comments to clarify what is deprecated.

**Review by jborean93 — APPROVED** [review]

**Pull request merged**